boards/arm/b_u585i_iot02a/b_u585i_iot02a.dts

```diff
@@ -17,6 +17,10 @@
 		zephyr,sram = &sram0;
 		zephyr,flash = &flash0;
 		zephyr,code-partition = &slot0_partition;
+		zephyr,bt-uart=&uart4;
+		zephyr,bt-c2h-uart=&uart4;
+		zephyr,bt-mon-uart=&uart4;
```

Comment:
Can you confirm using btmon is working with this setting ?

Reply:
Not confirmed. removed from the device tree.